project/plugins.sbt

```diff
@@ -1,7 +1,5 @@
-logLevel := Level.Warn
+//logLevel := Level.Warn
 
-addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.8.2")
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.5.2")
```

Comment:
downgrade?

Reply:
that was removal of scalariform, but I have now updated to 1.7.0